test/test.cli.ts

```diff
@@ -3,6 +3,7 @@ import * as execa from 'execa';
 import {assert} from 'chai';
 import * as http from 'http';
 import * as util from 'util';
+import stripAnsi = require('strip-ansi');
```

Comment:
My initial thought is that I would rather not introduce a new dependency (even in tests) if I don't have to :) What's going on here?  Why do we need this?

Reply:
https://github.com/JustinBeckwith/linkinator/runs/1677570613

`strip-ansi` is already in the deps tree anyway. Just do `npm ls strip-ansi`. And it's quite small anyway: https://packagephobia.com/result?p=strip-ansi

Reply:
I mean, there are only like 27 other dependencies that have this in the tree 😆   Fair enough.